docs/rotation.md

```diff
@@ -0,0 +1,21 @@
+## rotation
+
+rotation generates, extends, and syncs rotation schedules.
+
+### Synopsis
+
+rotation generates, extends, and syncs rotation schedules.
```

Comment:
```suggestion
`rotation` generates, extends, and syncs rotation schedules.
```

Reply:
Here you're saying the same thing as above. Maybe put usage here, with the whole command itself plus the options?